### DISCUSSION TITLE
[Vite:css] Lexical Error (unrecognised text)

### DISCUSSION THREAD

**Discussion by AuliaAmirullah15** [tailwindlabs/tailwindcss#13867] [Help]
Hi, I've followed this thread (https://github.com/tailwindlabs/tailwindcss/issues/1603 )to fix my issue which seems pretty similar with that thread, but I have no luck. I've defined spacing: { 'number': 'number'} in theme extend and variants extend, but they don't work at all. I assumed that might be related to my test file too, so I commented them out but still no luck. Does anyone know what the problem is, please?

```
  Erroneous area:
1: number * var(--tw-space-y-reverse)
^..^
1  |  @tailwind base;
   |  ^^^^^^^^^^^^^^^
2  |  
   |  ^
3  |  @tailwind components;
   |  ^^^^^^^^^^^^^^^^^^^^^
4  |  
   |  ^
5  |  @tailwind utilities;
   |  ^^^^^^^^^^^^^^^^^^^^
6  |  
   |  ^
```

**Comment by wongjn** [comment]
Could you provide a project that reproduces the unexpected behavior please?

**Reply by adamwathan** [reply]
It sounds like you are doing this:

```js
// tailwind.config.js
module.exports = {
  theme: {
    spacing: {
      number: 'number'
    }
  }
}
```

But `number` isn't a valid value for things like padding or anywhere where spacing is used:

```css
.pt-number {
  padding-top: number
}
```

It should be an actual number, like `20px`. Sounds like the error is because we are trying to do a `calc(…)` operation and the string "number" can't be used in `calc()`.

What are you actually trying to do?

**Reply by AuliaAmirullah15** [reply]
This error comes from tailwind. Seems number is undefined. I've tried putting that number definition in theme.spacing or theme.extend.spacing or removing it from those configs. But, with no luck.

**Reply by adamwathan** [reply]
Number isn't a variable though, it's a string:

> I've defined spacing: { 'number': 'number'}

Are you trying to use a variable called number? If so, you don't want quotes around the word `number` on the right hand side of the object.

**Reply by AuliaAmirullah15** [reply]
Yes, I've tried it too. But, it didn't work either. However, I have a suspicion over Vite for Unit Testing as the warning mentions [Vite:css] at the beginning of the warning. I'm starting to think we need to tell something about tailwind in vite config. But, not sure what it is.

**Reply by adamwathan** [reply]
Can you share reproduction so we can see the error happen on our own machines? I still don’t really understand what you’re doing.
